pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go

```diff
@@ -46,6 +46,7 @@ var (
 		"cluster-manager/0000_01_clusters.open-cluster-management.io_managedclustersetbindings.crd.yaml",
 		"cluster-manager/0000_03_clusters.open-cluster-management.io_placements.crd.yaml",
 		"cluster-manager/0000_04_clusters.open-cluster-management.io_placementdecisions.crd.yaml",
+		"cluster-manager/0000_05_clusters.open-cluster-management.io_addonplacementscores.crd.yaml",
```

Comment:
there are ut and integration test cases about the count of resources deployed by the operator, need to update them too.

Reply:
@zhiweiyin318 thanks for the comment, all the testing cases are updated.